### PR TITLE
Add AXI Lite registers automatic test

### PR DESCRIPTION
**Pull request by acomodi**
This PR builds on top of https://github.com/SymbiFlow/symbiflow-examples/pull/84 (and #83) and adds a simple AXI Lite test which aim is to provide a bitstream for the automatic zynq bitstream tester (https://github.com/SymbiFlow/symbiflow-xc7z-automatic-tester).

### CODE REVIEW

File: .github/workflows/tuttest.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Set Up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
```
Comment (mithro) [inline .github/workflows/tuttest.yml:56]: Can you split this into a separate pull request?

File: README.rst [hunk 2]
```diff
@@ -43,14 +43,14 @@ Install the following prerequisites before using symbiflow-examples:
 .. code:: bash
    :name: install-req-ubuntu
 
-   apt update && apt install -y wget
+   apt update && apt install -y wget curl
```
Comment (mithro) [inline README.rst:46]: Can you split this into a separate pull request?

File: README.rst [hunk 3]
```diff
@@ -91,10 +91,10 @@ and so you will need to add some ``sudo`` commands to the instructions below.
...
-        mkdir -p $INSTALL_DIR/xc7/install/share/symbiflow/arch
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/1049/20201123-030526/symbiflow-xc7a50t_test.tar.xz | tar -xJC $INSTALL_DIR/xc7/install/share/symbiflow/arch
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/1049/20201123-030526/symbiflow-xc7a100t_test.tar.xz | tar -xJC $INSTALL_DIR/xc7/install/share/symbiflow/arch
+        curl -s https://storage.googleapis.com/symbiflow-arch-defs-gha/symbiflow-toolchain-latest | xargs wget -qO- | tar -xJC $INSTALL_DIR/xc7/install
+        curl -s https://storage.googleapis.com/symbiflow-arch-defs-gha/symbiflow-xc7a50t_test-latest | xargs wget -qO- | tar -xJC $INSTALL_DIR/xc7/install
+        curl -s https://storage.googleapis.com/symbiflow-arch-defs-gha/symbiflow-xc7a100t_test-latest | xargs wget -qO- | tar -xJC $INSTALL_DIR/xc7/install
+        curl -s https://storage.googleapis.com/symbiflow-arch-defs-gha/symbiflow-xc7z010_test-latest | xargs wget -qO- | tar -xJC $INSTALL_DIR/xc7/install
```
Comment (mithro) [inline README.rst:97]: Can you split this into a separate pull request.

**Review by mithro — CHANGES_REQUESTED** [review]
This pull request has a bunch of things in it?

File: README.rst (hunk 2 above)
Comment (acomodi) [reply README.rst:46]: Given that I used curl to get the content of the latest-packages links, I think that this change and the change in the packages retrieval should go in the same PR (which is already open here: https://github.com/SymbiFlow/symbiflow-examples/pull/83)

**Comment by acomodi** [comment]
> This pull request has a bunch of things in it?

@mithro Yeah, this PR is based off other PRs, that is why it has many things. I will split changes in additional PRs though

**Comment by rw1nkler** [comment]
Please rebase the PR to enable documentation tests in the CI (building on RTD)

File: README.rst (hunk 3 above)
Comment (acomodi) [reply README.rst]: Done

File: README.rst (hunk 2 above)
Comment (acomodi) [reply README.rst]: In separate PR

File: .github/workflows/tuttest.yml (hunk 1 above)
Comment (acomodi) [reply .github/workflows/tuttest.yml]: Done in separate PR

**Review by mithro — APPROVED** [review]
If CI says "yes".

**Comment by mithro** [comment]
Is this pull request dead, should it be closed?

**Comment by acomodi** [comment]
I think that this one is not too relevant anymore to have in `symbiflow-examples` and can be closed. Might still be useful elsewhere (e.g. a future CI that has access to a zybo or similar). Closing
